docs/docs/20-quickstart.md

```diff
@@ -261,13 +261,15 @@ all three Argo CD `Application`s have not yet synced because they're not
 configured to do so automatically, and in fact, the branches referenced by their
 `targetRevision` fields do not even exist yet.
 
+![Argo-dashboard-screenshot](../static/img/argo-dashboard.png)
+
 :::info
 Our three stages all existing in a single cluster is for the sake of expediency.
 A single Argo CD control plane can manage multiple clusters, so these could just
 as easily have been spread across multiple clusters.
 :::
 
-### Hands on with the Kargo CLI
+### Define Your First Pipeline
```

Comment:
There's enough in both the YAML and explanatory text that follows that it might be better to make this heading slightly less specific:

```suggestion
### Your First Kargo Project
```